src/main/java/uk/gov/hmcts/reform/prl/services/citizen/CaseService.java

```diff
@@ -829,10 +832,37 @@ private List<CitizenNotification> getAllCitizenDashboardNotifications(CaseData c
         } else {
             citizenNotifications.add(CitizenNotification.builder().id(CAN_10_FM5).show(false).build());
         }
-
+        // CAN6,CAN6A,CAN6B
+        populateCitizenNotifications(
+            null != caseData.getRespondentResponseNotificationDetails()
+                && "YES".equalsIgnoreCase(caseData.getRespondentResponseNotificationDetails().getAp13NotificationSent()),
+            citizenNotifications,
+            CAN_6_AP_13
+        );
+        populateCitizenNotifications(
+            null != caseData.getRespondentResponseNotificationDetails()
+                && "YES".equalsIgnoreCase(caseData.getRespondentResponseNotificationDetails().getAp14NotificationSent()),
+            citizenNotifications,
+            CAN_6A_AP_14
+        );
+        populateCitizenNotifications(
+            null != caseData.getRespondentResponseNotificationDetails()
+                && "YES".equalsIgnoreCase(caseData.getRespondentResponseNotificationDetails().getAp15NotificationSent()),
+            citizenNotifications,
+            CAN_6B_AP_15
+        );
         return citizenNotifications;
     }
 
+    private void populateCitizenNotifications(boolean caseData, List<CitizenNotification> citizenNotifications,
+                                              String notificationType) {
+        if (caseData) {
```

Comment:
what is caseData? please give some meaningful name

Reply:
removed as discussed